.circleci/slackpost.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Usage: slackpost "<channel>" "<username>" "<message>"
+
+# ------------
+webhook_url="https://hooks.slack.com/services/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"
+
+# ------------
+channel=$1
+if [[ $channel == "" ]]
+then
+        echo "No channel specified"
+        exit 1
+fi
+
+# ------------
+shift
+username=$1
+if [[ $username == "" ]]
+then
+        echo "No username specified"
+        exit 1
+fi
+
+# ------------
+text="New deployment! Woo! $CIRCLE_PROJECT_USERNAME: $CIRCLE_PULL_REQUEST $CIRCLE_BRANCH $CIRCLE_TAG"
```

Comment:
$CIRCLE_BRANCH isn't set by CircleCI on tag builds. You can see my workaround for this in `run_terraform.sh` and `update_docker_img.sh`